shared/naturalcrit/splitPane/splitPane.jsx

```diff
@@ -1,200 +1,121 @@
 require('./splitPane.less');
 const React = require('react');
-const createClass = require('create-react-class');
+const { useState, useEffect, useRef } = React;
 const cx = require('classnames');
 
-const SplitPane = createClass({
-	displayName     : 'SplitPane',
-	getDefaultProps : function() {
-		return {
-			storageKey         : 'naturalcrit-pane-split',
-			onDragFinish       : function(){}, //fires when dragging
-			showDividerButtons : true
+const SplitPane = (props)=>{
+	const {
+		storageKey = 'naturalcrit-pane-split',
```

Comment:
storageKey doesn't need to be a prop. It is never passed in by any parent component.

Reply:
Alright, this is left from the original comp. resolved